io.asgardio.java.oidc.sdk/src/main/java/io/asgardio/java/oidc/sdk/SSOAgentConstants.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.asgardio.java.oidc.sdk;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This class holds the constants used in the module, io.asgardio.java.oidc.sdk.
+ */
+public class SSOAgentConstants {
+
+    public static final String CONFIG_BEAN_NAME = "io.asgardio.java.oidc.sdk.config.model.OIDCAgentConfig";
+
+    // Oauth response parameters and session attributes
+    public static final String ERROR = "error";
+    public static final String ACCESS_TOKEN = "access_token";
+    public static final String ID_TOKEN = "id_token";
+    public static final String SESSION_STATE = "session_state";
+    public static final String USER = "user";
+
+    // Keystore file properties
+    public static final String KEYSTORE_NAME = "keystorename";
+    public static final String KEYSTORE_PASSWORD = "keystorepassword";
+
+    // application specific request parameters and session attributes
+    public static final String CONSUMER_KEY = "consumerKey";
+    public static final String CONSUMER_SECRET = "consumerSecret";
+    public static final String CALL_BACK_URL = "callBackURL";
+    public static final String SKIP_URIS = "skipURIs";
+    public static final String INDEX_PAGE = "indexPage";
+    public static final String LOGOUT_URL = "logoutURL";
+    public static final String SCOPE = "scope";
+    public static final String OAUTH2_GRANT_TYPE = "grantType";
+    public static final String OAUTH2_AUTHZ_ENDPOINT = "authorizeEndpoint";
+    public static final String OIDC_LOGOUT_ENDPOINT = "logoutEndpoint";
+    public static final String OIDC_SESSION_IFRAME_ENDPOINT = "sessionIFrameEndpoint";
+    public static final String OIDC_TOKEN_ENDPOINT = "tokenEndpoint";
+    public static final String OIDC_ISSUER = "issuer";
+    public static final String OIDC_JWKS_ENDPOINT = "jwksEndpoint";
+    public static final String POST_LOGOUT_REDIRECTION_URI = "postLogoutRedirectURI";
+    public static final String AUTHENTICATED = "authenticated";
+    public static final String AUTHENTICATION_INFO = "authenticationInfo";
+    public static final String OIDC_OPENID = "openid";
+
+    // request headers
+    public static final String REFERER = "referer";
+
+    //context params
```

Comment:
Have a space before the beginning of the sentence

Reply:
Resolved with https://github.com/asgardio/asgardio-java-oidc-sdk/pull/1/commits/87be1ad8b44e103c81d6ce8a6ed22d4f6fc4f854.